### ISSUE TITLE
Release P-GenRM checkpoints on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [Tongyi-ConvAI/Qwen-Character#3]
Hi @tnlin 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2602.12116. Huge congrats on the ICLR 2026 oral!

The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), you can also claim the paper as yours which will show up on your public profile at HF, and add Github and project page URLs.

I saw that you've released the code for **P-GenRM** on GitHub. Would you like to host the pre-trained checkpoints on https://huggingface.co/models?

Hosting on Hugging Face will give your work more visibility and enable better discoverability. We can add tags in the model cards so that people find the models easier, link it to the paper page, and allow researchers to easily integrate your reward model into their alignment pipelines.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a custom PyTorch model, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to the model, which lets you to upload the model and people to download and use models right away.

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

Let me know if you're interested or need any guidance :)

Kind regards,

Niels
